src/openshift/component.ts

```diff
@@ -150,13 +150,44 @@ export class Component extends OpenShiftItem {
         );
     }
 
+    static async verbosityLevel() {
+        const level = [];
+        for (let i = 0; i <= 9; i++) {
+            level.push(i.toString());
+        }
+        return level;
+    }
+
+    static async showInformationMessage() {
+        return await window.showInformationMessage(`Do you want change verbosity level (Default value 0)`, 'Yes', 'No');
+    }
+
+    static async getVerbosityLevel() {
+        if (await Component.showInformationMessage() === 'Yes') {
+            const level = await window.showQuickPick(Component.verbosityLevel(), {placeHolder: "Select verbosity level"});
+            return level ? level : 0;
+        }
+        return 0;
+    }
+
+    static async verbosity(context: OpenShiftObject): Promise<string> {
+        const component = await Component.getOpenShiftCmdData(context,
```

Comment:
There is no reason to ask about Project/Application/Component. In first implementation we just set verbosity level for vscode workspace. That will affect all the commands.